src/main/java/frc/robot/subsystems/wrist/WristIO.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.subsystems.wrist;
+
+import org.littletonrobotics.junction.AutoLog;
+
+import frc.lib.advantagekit.LoggedIO;
+import frc.robot.subsystems.wrist.WristIO.WristPosition;
+
+public interface WristIO extends LoggedIO<WristPosition> {
+
+  @AutoLog
+  public static class WristPosition {
```

Comment:
Maintain a consistent naming convention for these classes. Should be `WristInputs`